epochalypse.py

```diff
@@ -123,7 +126,10 @@ def main():
     fromEpoch(float(args.epoch_input))
     print('')
   elif args.hexadecimal_input:
-    epoch = fromHex(args.hexadecimal_input)
+    hex_text = args.hexadecimal_input.replace(' ', '')
+    if args.revhex:
+      hex_text = binascii.hexlify( binascii.unhexlify( hex_text )[::-1] ).decode()
```

Comment:
Remove space within hexlify(), after the open brackets and before the closing ones